src/overcooked_ai_py/mdp/layout_generator.py

```diff
@@ -67,17 +67,18 @@ def __init__(self, params_schedule_fn=None, mdp_params_always=None):
         """
         assert params_schedule_fn != None or mdp_params_always != None, \
             "one of params_schedule_fn and mdp_params_always must be not Null"
-        self.mdp_params_always = mdp_params_always
-        self.params_schedule_fn = params_schedule_fn
+        if mdp_params_always != None:
```

Comment:
We don’t have to incorporate this into this review specifically (and in fact this is up to you), but what I was suggesting was to remove mdp_params_always entirely from the `__init__` signature: if you want to keep the ability to easily specify a params dict that stays constant over time, you could have a `classmethod` `.from_fixed_params(fixed_params)`, similar to `from_mdp(mdp)` for OvercookedEnv. I usually tend to like having the `__init__` logic and signatures as simple as possible, keeping the `__init__` signature have the most general case, and if necessary adding special-case logic through classmethods.

Reply:
I see what you mean. Sounds reasonable. This has been updated in the newest commit